### PR TITLE
[DONE] upgrade to django 1.10

### PR DESCRIPTION
**Pull request by larsclaussen**

### CODE REVIEW

**Comment by larsclaussen** [comment]
[django threedi-msr upgrade 1.6 --> 1.10 [2]](https://trello.com/c/Vcwvxyo2/992-django-threedi-msr-upgrade-1-6-1-10-2)

File: buildout.cfg [hunk 1]
```diff
@@ -26,12 +26,12 @@ eggs =
 # Specific pins. We inherit lots of pins from the KGS, so if you add a
 # development egg, you really need to un-pin it here.
 model-databank =
-Django = 1.6.5
-djangorecipe = 2.1.1
-zc.buildout = 2.4.0
-zc.recipe.egg = 2.0.2
+Django = 1.10
```
Comment (jsmits) [inline buildout.cfg]: -> 1.10.3

File: model_databank/testsettings.py [hunk 2]
```diff
@@ -41,38 +39,31 @@
...
-
+    'lizard_auth_client',
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.gis',
     'django.contrib.sites',
-    'django.contrib.webdesign',
+    #'django.contrib.webdesign',
```
Comment (jsmits) [inline model_databank/testsettings.py]: Kan weg.

File: model_databank/testsettings.py [hunk 3]
```diff
@@ -95,6 +86,8 @@
     'compressor.finders.CompressorFinder',
```
Comment (jsmits) [inline model_databank/testsettings.py]: Kan weg.

**Pull request merged**